micropub.php

```diff
@@ -142,28 +143,33 @@ public static function parse_query( $wp ) {
 	 */
 	private static function authorize() {
 		// find the access token
-		$auth = static::get_header( 'authorization' );
+		$auth  = static::get_header( 'authorization' );
```

Comment:
ugh. this variable init aligning is annoying when there are so few vars, like just two here. ugh.